src/platform/packages/private/kbn-index-editor/src/index_update_service.ts

```diff
@@ -392,6 +402,12 @@ export class IndexUpdateService {
     );
   }
 
+  private buildPlaceholderRow(): DataTableRecord {
+    return buildDataTableRecord({
+      _id: `${ROW_PLACEHOLDER_PREFIX}${this._rowPlaceholderCount++}`,
```

Comment:
Do we actually need a counter? It doesn't seem to be used. Shall we replace with a random string?